config/environments/production.rb

```diff
@@ -60,9 +60,9 @@
   # Use a different cache store in production.
   config.cache_store = :redis_cache_store, { url: ENV.fetch("REDIS_URL", "redis://localhost:6379/0") }
 
-  # Use a real queuing backend for Active Job (and separate queues per environment)
-  config.active_job.queue_adapter     = :sidekiq
-  # config.active_job.queue_name_prefix = "diffusion-marketplace_#{Rails.env}"
+  # Use a real queuing backend for Active Job (and separate queues per environment).
+  # config.active_job.queue_adapter     = :resque
+  # config.active_job.queue_name_prefix = "diffusion_marketplace_production"
```

Comment:
@PhilipDeFraties Can you reenable sidekiq and test in local?

Reply:
@camillevilla thanks for catching that, added it back and verified it works as expected when using the "Send Email to Innovation Editors" feature